gnovm/pkg/gnolang/gotypecheck.go

```diff
@@ -177,3 +180,27 @@ func deleteOldIdents(idents map[string]func(), f *ast.File) {
 		}
 	}
 }
+
+func filterCrossing(f *ast.File) {
+	astutil.Apply(f, nil, func(c *astutil.Cursor) bool {
+		switch n := c.Node().(type) {
+		case *ast.ExprStmt:
+			if ce, ok := n.X.(*ast.CallExpr); ok {
+				if id, ok := ce.Fun.(*ast.Ident); ok && id.Name == "crossing" {
+					// Delete statement 'crossing()'.
+					c.Delete()
+				}
+			}
+		case *ast.CallExpr:
+			if id, ok := n.Fun.(*ast.Ident); ok && id.Name == "cross" {
+				// Replace expression 'cross(x)' by 'x'.
+				var a ast.Node
+				if len(n.Args) > 0 {
+					a = n.Args[0]
+				}
```

Comment:
nitpick: len of n.Args can only be 1, right?

Reply:
Yes. I should probably validate the syntax of `cross()` and `crossing()` while they are still available for check!